src/NUnitConsole/nunit3-console.tests/ConsoleMocks.cs

```diff
@@ -1,5 +1,5 @@
 ﻿// ***********************************************************************
```

Comment:
This isn't a file move, GitHub has matched these two files up incorrectly.
- IConverter.cs was an unused class in the console
- ConsoleMocks.cs has been added to move test-only code out of the production project and into the test suite.